docs/UserGuide.md

```diff
@@ -87,22 +87,27 @@ Shows a list of all persons in the address book.
 
 Format: `list`
 
-### Editing a person : `edit`
 
-Edits an existing person in the address book.
+(Written by: Vaishak)
 
-Format: `edit INDEX [n/NAME] [p/PHONE] [e/EMAIL] [a/ADDRESS] [t/TAG]…​`
+### Editing a student : `edit`
 
-* Edits the person at the specified `INDEX`. The index refers to the index number shown in the displayed person list. The index **must be a positive integer** 1, 2, 3, …​
+Edits an existing student in Reeve.
+
+Format: `edit INDEX n/NAME p/PHONE s/SCHOOL y/YEAR v/VENUE t/TIME [a/ADDITIONAL_DETAILS] [m/MEETING_LINK] [sb/SUBJECT]`
+
+<div markdown="span" class="alert alert-primary">:information_source: **Note:**
+The format of TIME is {int: day_of_week} {int: start_time}-{int: end_time}
+</div>
+
+
+* Edits the student at the specified `INDEX`. The index refers to the index number shown in the displayed student list. The index **must be a positive integer** 1, 2, 3, …​
 * At least one of the optional fields must be provided.
 * Existing values will be updated to the input values.
-* When editing tags, the existing tags of the person will be removed i.e adding of tags is not cumulative.
-* You can remove all the person’s tags by typing `t/` without
-    specifying any tags after it.
 
 Examples:
-*  `edit 1 p/91234567 e/johndoe@example.com` Edits the phone number and email address of the 1st person to be `91234567` and `johndoe@example.com` respectively.
-*  `edit 2 n/Betsy Crower t/` Edits the name of the 2nd person to be `Betsy Crower` and clears all existing tags.
+*  `edit 1 n/Alex p/99999999 s/Meridian Junior College` Edits the name, phone number and school of the 1st person to be `Alex`, `99999999` and `Meridian Junior College` respectively.
+*  `edit 3 e/amongus@isgood.com sb/Mathematics v/Blk 33 West Coast Rd #21-214 t/1 1430-1630` Edits the email address, subject, venue and time of the third student to be `amongus@isgood.com`, `Mathematics`, `Blk 33 West Coast Rd #21-214` and `1 1430-1630` respectively.
```

Comment:
Might want to remove the "e/amongus@isgood.com" part

Reply:
Fixed, thanks!

Reply:
nice